modules/nw-externalip-about.adoc

```diff
@@ -51,7 +51,7 @@ If routed correctly, external traffic from the configured external IP address bl
 
 [IMPORTANT]
 ====
-You must ensure that the IP address block you assign terminates at one or more nodes in your cluster.
```

Comment:
unrelated to your PR, but this sentence is just confusing `terminates at one of more nodes in your cluster?` what does that mean?